tests/python_on_whales/components/buildx/test_buildx_cli_wrapper.py

```diff
@@ -654,6 +654,33 @@ def test_bake_with_variables_2(only_print, monkeypatch):
     }
 
 
+@pytest.mark.usefixtures("with_docker_driver")
+@pytest.mark.usefixtures("change_cwd")
+@pytest.mark.parametrize("only_print", [True, False])
+def test_bake_with_remote_context(only_print):
+    config = docker.buildx.bake(
+        print=only_print,
+        context="https://github.com/gabrieldemarmiesse/python-on-whales.git#v0.74.0:tests/python_on_whales/components/bake_tests",
+    )
```

Comment:
Can you also add a test where you specify a target to make sure the arguments are passed in the right order?

Reply:
Good call, [added](https://github.com/gabrieldemarmiesse/python-on-whales/pull/656/files#diff-e30722b601498d80a6278c8eb440452e25f646641ded295374cd13402ff58fd9R687)